Documentation/Helm-Charts/operator-chart.md

```diff
@@ -114,8 +114,6 @@ The following table lists the configurable parameters of the rook-operator chart
 | `csi.rbdPluginUpdateStrategy` | CSI RBD plugin daemonset update strategy, supported values are OnDelete and RollingUpdate | `RollingUpdate` |
 | `csi.rbdPluginUpdateStrategyMaxUnavailable` | A maxUnavailable parameter of CSI RBD plugin daemonset update strategy. | `1` |
 | `csi.rbdPodLabels` | Labels to add to the CSI RBD Deployments and DaemonSets Pods | `nil` |
-| `csi.readAffinity.crushLocationLabels` | Define which node labels to use as CRUSH location. This should correspond to the values set in the CRUSH map. | labels listed [here](../CRDs/Cluster/ceph-cluster-crd.md#osd-topology) |
```

Comment:
As we are removing so we wont need this https://github.com/rook/rook/issues/13243?

Reply:
This PR just removes the read affinity option from the rook operator config map. 
Not the read affinity feature.

Reply:
This PR is just removing settings that are unused since #13200, right?

Reply:
Yes, since we moved read affinity setting to cephCluster CR. 
No longer needed in operator config.

Reply:
Sorry now got a question what about the upgraded cluster which is still setting the value in ConfigMap but not in cluster CR?

Reply:
Will have to be migrated to cluster CR. 
Need to mention in Release notes.

Reply:
Can you please take care of it or else people will miss it, @travisn are we okay with this or need to keep this option to be backward compatible?

Reply:
Since v1.13 was released with #13200 (which moved the read affinity to the cluster CR), has the "--enable-read-affinity=true" flag been ignored by the CSI driver? Or has that flag still been honored even if there was a conflicting setting in the cluster CR? If this flag is already broken or ignored in v1.13, then the backward compatibility is already broken and we should go put a note in the v1.13 upgrade guide about this issue.

If we could maintain backward compatibility it would be ideal. IMO this feature is not critical to maintain backward compatibility, but if it's not broken yet and we're not going to maintain backward compatibility we should wait to release it in v1.14 and include a note in the upgrade guide. 

Please also split this out into a separate PR to give it more visibility.

Reply:
> has the "--enable-read-affinity=true" flag been ignored by the CSI driver? Or has that flag still been honored even if there was a conflicting setting in the cluster CR?

Its still honoured. cluster CR takes the precedence over `--enable-read-affinity=true` flag.
Will Split this to separate PR.

Reply:
> Please also split this out into a separate PR to give it more visibility.

PR https://github.com/rook/rook/pull/13665